metric/system/process/process_linux_test.go

```diff
@@ -97,3 +98,38 @@ func TestFetchProcessFromOtherUser(t *testing.T) {
 	require.NoError(t, err)
 	t.Logf("got: %s", pidData.StringToPrint())
 }
+
+func TestGetInfoForPid_NumThreads(t *testing.T) {
+	want := ProcState{
+		Name:       "elastic-agent",
+		State:      "sleeping",
+		Pid:        opt.IntWith(42),
+		Ppid:       opt.IntWith(1),
+		Pgid:       opt.IntWith(4067478),
+		NumThreads: opt.IntWith(26),
+	}
+
+	got, err := GetInfoForPid(resolve.NewTestResolver("testdata"), 42)
+	require.NoError(t, err, "GetInfoForPid returned an error when it should have succeeded")
+
+	assert.Equal(t, want, got)
+}
+
+func TestParsePIDInfos(t *testing.T) {
```

Comment:
I don't fully get the name of this test. If I understood it correctly, it's testing `parseProcStat`, however it is called `ParsePIDInfos`.
```suggestion
func TestParseProcStat(t *testing.T) {
```